dev-docs/modules/userid-submodules/hadron.md

```diff
@@ -38,17 +38,20 @@ The `request.userId.hadronId` will contain the Audigent HadronId:
-| params.partnerId | Required | Number | This is the Audigent Partner ID obtained from Audigent. |
+| Param under usersync.userIds[] | Scope    | Type    | Description                                                                                                                                                                                                                 | Example                                                                     |
+|--------------------------------|----------|---------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------|
+| name                           | Required | String  | ID value for the HadronID module - `"hadronId"`                                                                                                                                                                             | `"hadronId"`                                                                |
+| storage                        | Required | Object  | The publisher must specify the local storage in which to store the results of the call to get the user ID. This can be either cookie or HTML5 storage.                                                                      |                                                                             |
+| storage.type                   | Required | String  | This is where the the user ID will be stored. The recommended method is `localStorage` by specifying `html5`.                                                                                                               | `"html5"`                                                                   |
+| storage.name                   | Required | String  | The name of the cookie or html5 local storage where the user ID will be stored. The recommended value is `hadronId`.                                                                                                        | `"auHadronId"`                                                              |
+| storage.expires                | Optional | Integer | How long (in days) the user ID information will be stored. The recommended value is 14 days.                                                                                                                                | `14`                                                                        |
+| value                          | Optional | Object  | Used only if the page has a separate mechanism for storing the Hadron ID. The value is an object containing the values to be sent to the adapters. In this scenario, no URL is called and nothing is added to local storage | `{"hadronId": "0aRSTUAackg79ijgd8e8j6kah9ed9j6hdfgb6cl00volopxo00npzjmmb"}` |
```

Comment:
This really long hadronId example makes the param table look bad. see https://deploy-preview-5693--prebid-docs-preview.netlify.app/dev-docs/modules/userid-submodules/hadron.html

I recommend replacing it with "0a...mb"

Reply:
thank you, will fix that

Reply:
@bretg done!

Reply:
thanks - preview looks better now.